scripting/league_voting.sp

```diff
@@ -91,3 +157,28 @@ public Action Timer_PreOT(Handle timer) {
     menu.ExitButton = false;
     menu.DisplayVoteToAll(20);
 }
+
+public Action Event_RoundEnd(Handle event, const char[] name, bool dontBroadcast) {
+    int roundsPlayed = GameRules_GetProp("m_totalRoundsPlayed");
+    int maxrounds = GetCvarIntSafe("mp_maxrounds");
+    Get5State state = Get5_GetGameState();
+    g_bIsOvertime = false;
+
+    if (CS_GetTeamScore(CS_TEAM_T) == CS_GetTeamScore(CS_TEAM_CT) && roundsPlayed == maxrounds) {
+        g_bIsOvertime = true;
+    }
+    
+    if (g_bIsOvertime && state == Get5State_Live) {
+        CreateTimer(0.1, Timer_PreOT);
+    }
+}
+
+public Action Command_Surrender(int client, int args) {
+    int CTScore = CS_GetTeamScore(CS_TEAM_CT);
+    int TScore = CS_GetTeamScore(CS_TEAM_T);
+    if (CTScore - 8 >= TScore || TScore - 8 >= CTScore) {
```

Comment:
I've seen this check a lot. Perhaps set it as a stock bool somewhere to prevent the duplication of logic?

Reply:
Stock now created.